src/datasets/schemas/dataset.schema.ts

```diff
@@ -453,44 +451,47 @@ export class DatasetClass extends OwnableClass {
   @ApiProperty({
     type: String,
     required: false,
-    description: "The ID of the proposal to which the dataset belongs.",
+    description:
+      "The ID of the proposal to which the dataset belongs to and it has been acquired under.",
   })
   @Prop({ type: String, ref: "Proposal", required: false })
   proposalId?: string;
 
   @ApiProperty({
-    type: String,
+    type: [String],
     required: false,
-    description: "ID of the sample used when collecting the data.",
+    description:
+      "Single ID or array of IDS of the samples used when collecting the data.",
   })
-  @Prop({ type: String, ref: "Sample", required: false })
-  sampleId?: string;
+  @Prop({ type: [String], ref: "Sample", required: false })
+  sampleId?: string[];
 
   @ApiProperty({
-    type: String,
+    type: [String],
     required: false,
-    description: "ID of the instrument where the data was created.",
+    description:
+      "Id of the instrument or array of IDS of the instruments where the data contained in this dataset was created/acquired.",
   })
-  @Prop({ type: String, ref: "Instrument", required: false })
-  instrumentId?: string;
+  @Prop({ type: [String], ref: "Instrument", required: false })
+  instrumentId?: string[];
```

Comment:
We need to change the naming to `sampleIds` and `instrumentIds` to avoid confusion